rero/rero-ils-ui
# general: add current library switch for system librarian

* Adds library switch service.
* Adds library service.
* Adds new parameter sessionExpiredSeconds on application configuration.
* Adds current library name on template for system librarian.
* Improves user service.

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

## Why are you opening this PR?

- Adds possibility for system librarian to switch current library to another.

## How to test?

- Login with system librarian
- Check menu "Switch libraries" on homepage
- Check info on top of page (library name)

## Code review check list
